kubectl-rook-ceph.sh

```diff
@@ -246,6 +250,15 @@ function run_rook_cr_status() {
   fi
 }
 
+function run_purge_osd() {
+   if [ "$#" -eq 1 ]; then
+    run_ceph_command osd purge osd."$1" --yes-i-really-mean-it --force
```

Comment:
Instead of running the ceph purge command, let's just run the `rook ceph osd remove` command, the same as the osd purge job. Then it will delete the osd deployment and all other related resources such as PVCs.

Reply:
good suggestion. 

changes done.